examples/core_api/argument_sets/time_lookup_argument_set.rb

```diff
@@ -1,11 +1,14 @@
 # frozen_string_literal: true
 
+require_relative "object_lookup"
+
 module CoreAPI
   module ArgumentSets
     class TimeLookupArgumentSet < Apia::LookupArgumentSet
 
       argument :unix, type: :string, required: true
       argument :string, type: :string
+      argument :nested, type: ObjectLookup
```

Comment:
I added the `ObjectLookup` argument here to reproduce the exception in our specs